src/test/java/cloud/eppo/EppoValueTest.java

```diff
@@ -0,0 +1,14 @@
+package cloud.eppo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import cloud.eppo.ufc.dto.EppoValue;
+import org.junit.jupiter.api.Test;
+
+public class EppoValueTest {
```

Comment:
Made by Eric in 2022 for android and still kicking ([source](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/test/java/cloud/eppo/android/EppoValueTest.java)) 🤷‍♂️